pkg/cmd/gist/list/list.go

```diff
@@ -33,17 +37,46 @@ func NewCmdList(f *cmdutil.Factory, runF func(*ListOptions) error) *cobra.Comman
 
 	var flagPublic bool
 	var flagSecret bool
+	var flagFilter string
 
 	cmd := &cobra.Command{
-		Use:     "list",
-		Short:   "List your gists",
+		Use:   "list",
+		Short: "List your gists",
+		Long: heredoc.Docf(`
+			List gists from your user account.
+
+			You can use a regular expression to filter the description, file names,
+			or even the content of files in the gist. See https://pkg.go.dev/regexp/syntax
+			for the regular expression syntax you can pass to %[1]s--filter%[1]s. Pass
+			%[1]s--include-content%[1]s to also search the content of files noting that
+			this will take longer since all files' content is fetched.
```

Comment:
At first, I was only only going to suggest the URL format change, so it is linkable on the CLI manual website.  Then I thought I could simplify the text a little.

```suggestion
			List gists from your user account.

			You can use a regular expression to filter the description, file names,
			or even the content of files in the gist using %[1]s--filter%[1]s.
			
			Use %[1]s--include-content%[1]s to include content of files, noting that
			this will be slower and increase rate limit used.

			For supported regular expression syntax, see <https://pkg.go.dev/regexp/syntax>
```

Reply:
Sure. Text looks good, but the `<>` around the link - while good for markdown - actually renders literally to the console. Other links in the CLI - like the footer of every command - don't have that, so I'll elide it; however, if you have reason to include that, should I do a separate PR to update all URLs in the CLI? FWIW, I don't think it adds anything because terminals' automatic hyperlinking still cares about continuity so any line breaks - regardless of surrounding characters - mess up hyperlinks unless explicitly hyperlinked via the OSC sequence we discussed way back and agreed it's not supported in enough terminals to use (sadly).

Reply:
Updated, but I'll leave this unresolved in case you wanted to respond to my comment above.